01/app.js

```diff
@@ -1 +1,13 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+
+const commentsElement = document.querySelector(
+	".comments__item",
+	".comments__item--newest"
```

Comment:
Nie powinnyśmy tego rozdzielać - prawidłowe rozwiązanie to `".comments__item.comments__item--newest"`